torchvision/features/_image.py

```diff
@@ -0,0 +1,11 @@
+from typing import Tuple
+
+from ._core import TensorFeature
+
+__all__ = ["Image"]
+
+
+class Image(TensorFeature):
```

Comment:
Nice. Maybe on the future that could include information like PIL's `mode` to know the type of the image.

Reply:
I think `mode` is a little overkill, since `PIL` also uses it to differentiate between integral and floating point images. This information is already encoded in the `dtype`. Still, something like `color_space` will be a great addition.

Reply:
I'm not arguing to add it. I said it opens up possibilities.